src/App.js

```diff
@@ -1006,6 +1009,11 @@ export class App extends React.Component<AppProps, AppState> {
         this.setState({keyBindingsEnabled: keyBindingsEnabled});
     }
 
+    handleShareButtonShortcut = () => {
+        const currentUrl = document.location.href;
+        return navigator.clipboard.writeText(currentUrl);
```

Comment:
Does this mean there's no onscreen or announced feedback?   It seems like there should be.

Reply:
Didn't we say that the shortcut will just copy the url to the clipboard without the feedback message? And communicate more clearly with the name used in grid key for the switches, like "copy the current url"?

Reply:
I still like letting people know that they've actually succeeded in hitting the key combination.  It doesn't have to be visual feedback, though.  Maybe this is a good use case for a preview announcement?

Reply:
I agree that a preview(maybe feedback you mean?) announcement can make this clear.

Reply:
Yes, I'm still getting them straight in my head, I guess in this case I did mean feedback.